dub.sdl

```diff
@@ -3,4 +3,4 @@ description "Extracted std.experimental.allocator"
 authors "Team Phobos"
 copyright "Copyright © 2017, Team Phobos"
 license "BSL-1.0"
-dependency "mir-core" version="~>0.0.5"
+dependency "mir-core" version=">0.0.5 <0.3.0"
```

Comment:
why < 0.3.0?

Reply:
The current version is v0.2.0, `< 0.3.0` include all v0.2.x, and not include 0.3.0

Reply:
Yes, but so would `~>0.2.0`.

https://dub.pm/package-format-sdl.html#version-specs

Reply:
The current change is `>0.0.5 <0.3.0` the `~>0.2.0` is different and does not include older versions. BTW, it should be `>=0.0.5 <0.3.0` to support older configs, will change it